app/src/main/java/com/github/se/studybuddies/ui/chat/ChatScreen.kt

```diff
@@ -504,3 +597,115 @@ fun SendPhotoMessage(messageViewModel: MessageViewModel, showAddImage: MutableSt
         })
   }
 }
+
+@Composable
+fun SendLinkMessage(messageViewModel: MessageViewModel, showAddLink: MutableState<Boolean>) {
+  val linkState = remember { mutableStateOf("") }
+  val linkName = remember { mutableStateOf("") }
+
+  if (showAddLink.value) {
+    AlertDialog(
```

Comment:
What I understand between AlertDialog and Dialog, it's that AlertDialog are easier to implement, but doesn't allow for more advance customizations.
So I choose the simplier way because I don't need the extra customization